docs/source/en/_toctree.yml

```diff
@@ -23,7 +23,7 @@
     title: Load and train adapters with 🤗 PEFT
   - local: model_sharing
     title: Share your model
-  - local: transformers_agents
+  - local: agents
```

Comment:
If at all possible it'd be nice to keep the old name here as if it's linked from elsewhere it would lead to a 404 post-rename

Reply:
Thank you! I just changed this back.